pkg/koordlet/metriccache/metric_types.go

```diff
@@ -89,6 +111,9 @@ var MetricPropertiesFunc = struct {
 			MetricPropertyGPUDeviceUUID: uuid,
 		}
 	},
+	PodContainer: func(podUID, containerID string) map[MetricProperty]string {
```

Comment:
we can define PodPSI/ContainerPSI func like:
func(id, resource_type, precision, degree)